ARE/tools/APE/src/main/java/eu/asterics/ape/main/APEProperties.java

```diff
@@ -43,6 +43,7 @@
     public static final String P_APE_PROJECT_DIR = "APE.projectDir";
     public static final String P_APE_BUILD_DIR = "APE.buildDir";
     public static final String P_APE_DATA_COPY_MODE = "APE.dataCopyMode";
+    public static final String P_APE_WEB_COPY_MODE = "APE.webCopyMode";
```

Comment:
is it also possible in ARE (RestServer) to use this AREProperties class? because then I could add a REST-method that retrieves the current APE-application version and use it for the AsTeRICS-Ergo footer -> then I would't have to edit this file each time on a new version.

Reply:
This class is APEProperties and is only available during an APE run.
The file APE.properties also won't be copied to the deployment directory.